pom.xml

```diff
@@ -262,6 +262,14 @@
     <version.javax.xml.ws>2.3.1</version.javax.xml.ws>
     <version.javax.annotation-api>1.3.2</version.javax.annotation-api>
     <version.com.sun.xml.messaging.saaj>1.5.0</version.com.sun.xml.messaging.saaj>
+    
+    <!-- specifal versions to support different version of hibernate for ee7 servers such as WAS and WLS -->
+    <version.org.hibernate4ee7>5.1.15.Final</version.org.hibernate4ee7>
+    <version.org.jboss.logging4ee7>3.3.1.Final</version.org.jboss.logging4ee7>
+    <version.org.javassist4ee7>3.20.0-GA</version.org.javassist4ee7>
+    <version.antlr4ee7>2.7.7</version.antlr4ee7>
+    <version.org.jboss.jandex4ee7>2.0.3.Final</version.org.jboss.jandex4ee7>
+    <version.com.fasterxml.classmate4ee7>1.3.0</version.com.fasterxml.classmate4ee7>
```

Comment:
@mswiderski thanks for adding new set of properties to differentiate them against the standard EE8. One minor note to property names, it could be probably easier to read to divide with dot or dash the last part `4ee7` in all of them.

Reply:
updated to use -